codecov.yml

```diff
@@ -1,2 +1,15 @@
 github_checks:
     annotations: false
+
+ignore:
```

Comment:
Would it make sense to add `manage.py`, `setup.py`, and `wsgi.py` as well?

Reply:
Ok I will remove this and we can omit it in `setup.cfg` instead.